packages/@icon-magic/logger/README.md

```diff
@@ -0,0 +1,51 @@
+# @icon-magic/logger
+
+---
+
+This package is responsible for exposing all the logging functionality for
+icon-magic. It contains a wrapper around winston.js for including the filename
+from where the logs are generated as a label in the logged message.
+
+Logs are written to the following simulataneously:
+
+1. console - logs of all levels
```

Comment:
Maybe a little out of scope for right now — any thoughts on making this configurable?

Reply:
Yes, I've created an issue to track this: https://github.com/linkedinlabs/icon-magic/issues/14